### DISCUSSION TITLE
Booting from SD card with wiped NAND / restoration process

### DISCUSSION THREAD

**Discussion by samey3** [xjtuecho/EBAZ4205#55] [Troubleshooting]
Hello,
I'm still unfamiliar with how the Zynq works, and I was wondering about the boot / potential board recovery processes.

I previously had it boot from an image on an SD card once before encountering errors and subsequently stopped working (I did not record them at the time). Following this I was playing around with it to see if I could correct it, and wiped the NAND flash at some point. Of course, there is now no output via the UART interface.

I'm wondering what would the recovery process entail here (ignoring the original SD issue)? Can it boot purely from a BOOT.BIN file on an SD card with nothing on the NAND flash (I assume this is the case), or must I deal with that first? Using Vivado/Vitis, I managed to create a new FSBL and flash it in, but as expected it did not solve my issue. 

Alternatively, has anyone created a set of files that I may flash to the memory to restore a version of petalinux/the original functionality?

Any help would be appreciated! I've looked around a fair bit/tried a few things but I'm in general unsure of what needs to be done

**Comment by DavidJRichards** [comment]
To restore the nand files will require booting from a sdcard with a working uboot system.

**Comment by jcdevel** [comment]
Is it really possible to boot from SD card? As per UG585:
"Note: The SD card boot mode is not supported in 7z010 dual core"
However, information about ebaz4205 (wikis, posts, etc) indicates that it is actually possible by setting the BOOT Select Resistors.
Can someone confirm if booting from SD card works? (I have not tried myself because I do not have a soldering pen to move R2577 to R2584)

**Reply by nic3-14159** [reply]
Yes, SD card boot works once you change the BOOT Select Resistors. The full sentence in UG585 is this: "The SD card boot mode is not supported in 7z010 dual core and 7z007s single core CLG225 devices". I suppose that could be interpreted as the CLG225 part only applying to the 7z007s, but the intended meaning is that both the CLG225 7z010 and CLG225 7z007s chips do not support the SD card boot mode. This is because those chips physically do not have the pins used by the bootrom for SD boot available. The 7z010 chip used on the EBAZ4205 uses the CLG400 package so the mentioned note does not apply.

**Reply by jcdevel** [reply]
Thank you very much for the quick response! :-)
